src/test/resources/logback.xml

```diff
@@ -1,18 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <appender name="STDOUT"
```

Comment:
1. Please keep this one consistent with the other ones. You generally don't need it; only for debugging purposes. Just to give you a reference: https://github.com/DANS-KNAW/easy-ingest/blob/master/src/test/resources/logback.xml
2. Let's keep the pattern the same as all other patterns in the same file across all EASY modules
3. Keep the log level configurable as it was; this way we can turn it on only when we need the logging

Reply:
The reason to write into a file is to be able to Unit test the log file.